res/datasets/dumpMaker.py

```diff
@@ -2,17 +2,18 @@
 
 import pandas as pd
 
-df = pd.read_csv('coin_Bitcoin.csv')
+df: pd.DataFrame = pd.read_csv("coin_Bitcoin.csv")
 
-values = []
+values: list[str] = []
 
-for date, high, low, _open, _close in zip(df['Date'], df["High"], df["Low"], df["Open"], df["Close"]):
+for date, high, low, _open, _close in zip(
+    df["Date"], df["High"], df["Low"], df["Open"], df["Close"]
+):
     date = date[:10]
-    value = sum((high, low, _open, _close))/4
+    value = sum((high, low, _open, _close)) / 4
     values.append(f"('{date}', {value})")
-    
-with open("dump.sql", "w") as f:
+
+with open(file="dump.sql", mode="w", encoding="utf-8") as f:
```

Comment:
W pythonie przekazuje się parametry przez nazwy, jeśli nie występują one po kolei. Tu nie ma takiej potrzeby dla file i mode. W celu zachowania spójności, trzeba by tak albo robić za każdym razem, albo tylko jeśli trzeba

Reply:
Keyword arguments named explicitly provide an easier way for developers to understand what these arguments mean. Python is not ideal, and simply adding `"w"` as argument does not say much.

Reply:
W porządku, jednak podawanie "file=" w przypadku open (bardzo znanej funkcji) to moim zdaniem lekka przesada.

Reply:
I guess it won't hurt anyone if it stays like this?